source/documentation/common/upgrade/proc-Upgrading_the_Manager_to_4-4.adoc

```diff
@@ -62,10 +63,33 @@ ifdef::local_database_upgrade,remote_database_upgrade[. Log in to the {engine-na
 ----
 # engine-backup --scope=all --mode=backup --file=backup.bck --log=backuplog.log
 ----
-
 . Copy the backup file to a storage device outside of the {virt-product-shortname} environment.
 
 ifdef::SHE_upgrade[]
+
+. Shut down the self-hosted engine.
++
+[options="nowrap" subs="normal"]
+----
+# poweroff
```

Comment:
You use poweroff here, but you say to shut down the machine. There's a significant difference between the two, and I think you mean shut down, so the command should not be poweroff. See https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html/virtual_machine_management_guide/chap-administrative_tasks

Reply:
The Upgrade Helper specifically gives the CLI command as # poweroff, but refers to the action as "shutdown"
The VMM guide, when talking about Shutting Down  a Virtual Machine (chap 6.1), refers to both "Shutdown" and "Power Off" options in the Admin Portal /VM Portal UI.

Reply:
Then ask Marina why. It's not correct to tell somebody to unplug the computer to turn it off, but that's what the poweroff command does. You should only use it when shutdown doesn't work.

Reply:
@emarcusRH Please change this as noted above. The Upgrade Helper will change the wording as well.